doc/classes/Node3D.xml

```diff
@@ -136,85 +143,91 @@
 		<method name="orthonormalize">
 			<return type="void" />
 			<description>
-				Resets this node's transformations (like scale, skew and taper) preserving its rotation and translation by performing Gram-Schmidt orthonormalization on this node's [Transform3D].
+				Orthonormalizes this node's [member basis]. This resets the [member scale] or any other distortion to the default, but preserves the [member position] and [member rotation]. See also [method Transform3D.orthonormalized].
 			</description>
 		</method>
 		<method name="rotate">
 			<return type="void" />
 			<param index="0" name="axis" type="Vector3" />
 			<param index="1" name="angle" type="float" />
 			<description>
-				Rotates the local transformation around axis, a unit [Vector3], by specified angle in radians.
+				Rotates this node's local [member transform] around the [param axis] by the given [param angle], in radians. The operation happens in local space.
```

Comment:
From how I understand it, `rotate` and `rotate_[xyz]` specify the axis in global space (parent space? not sure about proper terminology), but both modify the local space `transform`. Whereas `rotate_object_local` also transforms the rotation axis into local space. Right now, both of these methods have the same description.

Reply:
I have kept these in mind for further testing. These methods have also always confused me so it would be nice to finally write something easier to understand.

Reply:
Just to give closure, `rotate` is indeed in parent space (same as the `transform` property). If the `position` was anything but `Vector3(0, 0, 0)`, the node would rotate _and_ move around the parent.